romana-install/config.yml

```diff
@@ -0,0 +1,36 @@
+---
+# Ensure stack nodes are ready for installation
+- hosts: stack_nodes
+  tasks:
+    - name: Wait for cloud-init to complete
+      wait_for: path="/var/lib/cloud/instance/boot-finished" state=present
+
+# Apply common settings to all nodes prior to installation
+- hosts: stack_nodes
+  roles:
+    - system/ubuntu
+
+# Include stack playbooks. Only the configured stack will be installed.
+- include: config.devstack.yml
```

Comment:
I'd attempted that, and it didn't behave as expected.
There's a difference between task-level includes and playbook-level includes. Task-level ones evaluate "when" items, and playbook-level ones ignore it.
This is my workaround for it instead of maintaining parallel playbooks for each stack type that differ by only a few stack-specific tasks.